core/templates/pages/story-viewer-page/story-viewer-page.component.spec.ts

```diff
@@ -79,10 +81,12 @@ describe('Story Viewer Page component', function() {
       'clasroom_1');
     spyOn(urlService, 'getStoryUrlFragmentFromLearnerUrl').and.returnValue(
       'story_1');
-    spyOn(userService, 'getUserInfoAsync').and.returnValue($q.resolve({
-      isLoggedIn: () => true
-    }));
-    spyOn(userService, 'getLoginUrlAsync').and.returnValue($q.resolve('/home'));
+    spyOn(userService, 'getUserInfoAsync')
+      .and.returnValue($q.resolve({
+        isLoggedIn: () => true
+      }));
+    spyOn(userService, 'getLoginUrlAsync')
+      .and.returnValue($q.resolve('/home'));
```

Comment:
Is this an expected change?

Reply:
Since "UserService" is name of a class being imported , I have changed the variable name to avoid naming clash !

Reply:
Thanks for the explanation, @subedi90. :) 

I was only expecting changes for the variable name. (Let's try to avoid extra changes in the PR!)

Reply:
done